misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/FormatUtil.java

```diff
@@ -117,6 +108,11 @@ static void execute(List<String> argList, boolean helpFlag, boolean dryRun, Path
                 } else {
                     moduleName = argList.get(0);
 
+                    // Check whether the given directory is not in a ballerina project.
+                    if (!FormatUtil.isBallerinaProject(sourceRootPath)) {
```

Comment:
We dont need this check here. `BuildProject.load()` will do the validation

Reply:
Done